samples/simple-client/src/main/java/com/squareup/okhttp/sample/OkHttpContributors.java

```diff
@@ -26,16 +27,19 @@ public static void main(String... args) throws Exception {
     OkHttpClient client = new OkHttpClient();
 
     // Create request for remote resource.
-    Request request = new Request.Builder()
-        .url(ENDPOINT)
+    Request request = new Request.Builder() //
+        .url(ENDPOINT) //
         .build();
 
     // Execute the request and retrieve the response.
     Response response = client.newCall(request).execute();
 
     // Deserialize HTTP response to concrete type.
-    Reader body = response.body().charStream();
-    List<Contributor> contributors = GSON.fromJson(body, CONTRIBUTORS.getType());
+    ResponseBody body = response.body();
+    Reader charStream = body.charStream();
+    List<Contributor> contributors = GSON.fromJson(charStream, CONTRIBUTORS.getType());
+    charStream.close();
```

Comment:
The `Reader` is just wrapping the `BufferedSource` and it holds no resources itself.